openshift/cm.yml

```diff
@@ -293,3 +245,67 @@ objects:
       spring.mail.password=${PROV_APP_MAIL_PASSWORD}
       provison.mail.sender=provision@${PROV_APP_MAIL_HOST}
       spring.main.allow-bean-definition-overriding=true
+
+- kind: ConfigMap
+  metadata:
+    labels:
+      app: prov-app
+    name: quickstarters.properties
```

Comment:
You'd then need to add `preserve bc:quickstarters.properties:/data` to the `Tailorfile` so no drift is generated for this.